### ISSUE TITLE
dm:hull isn't working

### ISSUE THREAD

**Issue by LinkFly** [Shirakumo/radiance-tutorial#1]
I'm doing by tutorial:
(dm:hull 'plaster-pastes)
After that error appears:

Condition of type DATABASE:INVALID-COLLECTION.
Cannot create hull.

**Comment by Shinmera** [comment]
I believe that's a bug that I recently fixed. If you update `(ql:update-dist "shirakumo")` and reload you should be fine.

**Comment by Shinmera** [comment]
Closing for inactivity.

**Comment by abishek** [comment]
I face this same issue. I have the lastest dist already.

I think the db:connected trigger is not occurring when I try to restart radiance (new session because my slime connection just crashed out of the blue). I am able to fetch existing records, so database connection itself exists. Only the trigger hasn't occurred and hence probably the 'plaster-pastes symbol is unknown. Its a guess since am a beginner in lisp and radiance.

**Comment by abishek** [comment]
I did some digging because I couldn't get past this (and I really like this framework over the others I've tried, thanks!). After going over the radiance-contribs code, I found a function `db:collections` which lists all the available collections for the active module (or session?). It listed my collection as `'radiance-user::my-collection-name`. I replaced the code with this qualified name and it doesn't fail anymore. I hope this helps.
